src/shared/utils/helpers/format-past-date.ts

```diff
@@ -2,8 +2,23 @@ import formatDistanceStrict from "date-fns/formatDistanceStrict";
 import parseISO from "date-fns/parseISO";
 
 export default function (dateString?: string) {
-  const parsed = parseISO((dateString ?? Date.now().toString()) + "Z");
-  return formatDistanceStrict(parsed, new Date(), {
-    addSuffix: true,
-  });
+  if (!dateString || dateString === undefined) {
+    console.error(err);
+    return "DATE ERROR";
+  }
+
+  try {
+    const parsed = parseISO((dateString ?? Date.now().toString()) + "Z");
```

Comment:
Since you return an error string if `dateString` is nullish, the null coalesce operator shouldn't be needed anymore:
```suggestion
    const parsed = parseISO(Date.now().toString() + "Z");
```